formats/json-tests/commonTest/src/kotlinx/serialization/test/CompilerVersions.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2017-2023 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.serialization.test
+
+import kotlin.test.Test
+import kotlin.test.assertFalse
+import kotlin.test.assertTrue
+
+private val currentKotlinVersion = KotlinVersion.CURRENT
+
+public fun runSince(kotlinVersion: String, test: () -> Unit) {
```

Comment:
I'd move this declaration down so all `private` declarations would go first in file to maintain the logical order

Also, `public` is unnecessary here as these are test sources. Either add `public` to `CompilerVersionTest` to maintain consistency or remove this one